src/integration-test/java/com/commercetools/sync/inventories/InventoryUpdateActionUtilsItTest.java

```diff
@@ -184,8 +184,13 @@ public void buildCustomUpdateActions_ShouldBuildActionThatSetCustomField() {
         final InventoryEntry oldInventoryBeforeSync = oldInventoryOptional.get();
         assertThat(oldInventoryBeforeSync.getCustom()).isNull();
 
+        //Fetch type and ensure it is present.
+        final Optional<Type> type = getTypeByKey(CTP_TARGET_CLIENT, CUSTOM_TYPE);
+        assertThat(type).isNotEmpty();
+        assertThat(type.get().getKey()).isEqualTo(CUSTOM_TYPE);
```

Comment:
Noo, here I fetch a type from CTP. I set the entire `Type` here https://github.com/commercetools/commercetools-sync-java/blob/inventory_sync_integration_tests/src/integration-test/java/com/commercetools/sync/inventories/InventoryUpdateActionUtilsItTest.java#L193